modules/speed.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Module Name: Speed
+ * Module Description: Increase the overall performance of WordPress. Better than caching.
+ * Jumpstart Description: Removes slow filters from WordPress.
+ * Sort Order: 1
+ * Recommendation Order: 1
+ * First Introduced: 4.0
+ * Requires Connection: No
+ * Auto Activate: Yes
```

Comment:
Should we auto activate and jumpstart that module? Even if you don't change the default speed value, it does change things on the frontend of your site (remove the missing capital P from WordPress).